templates/compositions/ec-component-highlighted-news-block/highlighted-news-block.html.twig

```diff
@@ -0,0 +1,43 @@
+{% apply spaceless %}
+{#
+  @file
+  Theme implementation for the Highlighted News Block
+  Parameters:
+    - "title" (string) (default: ''): The Highlighted News Block title.
+    - "highlighted" (array) (default: []): The main item of the list compatible with the ECL Content Item component.
+    - "items" (array) (default: []): Highlighted News items compatible with the ECL Content Item component.
+    - "see_more" (array) (default: []): Predefined structure compatible with Link component.
+    - "detail" (string) (default: ''): Additional information to be displayed below the 'See more' link.
+#}
+  <div id="highlighted-news-block">
```

Comment:
Is it safe to use hard-coded id?

Reply:
I'd say so, there is no JS, what would be the concern?

Reply:
Just for keeping validity of html.